src/lib/ledger_proof/ledger_proof_intf.ml

```diff
@@ -16,19 +16,28 @@ module type S = sig
   end]
 
   val create :
-       statement:Transaction_snark.Statement.t
+       statement:Mina_state.Snarked_ledger_state.t
     -> sok_digest:Sok_message.Digest.t
     -> proof:Proof.t
     -> t
 
   val statement_target :
```

Comment:
should this function be renamed, given its new input type?

Reply:
I only moved the module to a different library, it is still the statement of a ledger proof